packages/jellyfish-api-core/__tests__/category/misc/signMessageWithPrivKey.test.ts

```diff
@@ -0,0 +1,37 @@
+import { MasterNodeRegTestContainer } from '@defichain/testcontainers'
+import { ContainerAdapterClient } from '../../container_adapter_client'
+import { RpcApiError } from '@defichain/jellyfish-api-core'
+
+describe('Verify message', () => {
+  const container = new MasterNodeRegTestContainer()
+  const client = new ContainerAdapterClient(container)
+
+  beforeAll(async () => {
+    await container.start()
+    await container.waitForReady()
```

Comment:
nitpick: `waitForReady()` is deprecated `start()` will automatically wait for ready